app/controllers/tracks_controller.rb

```diff
@@ -0,0 +1,47 @@
+class TracksController < ApplicationController
+  load_resource :conference, find_by: :short_title
+  load_resource :program, through: :conference, singleton: true
+  load_and_authorize_resource through: :program, find_by: :short_name
+
+  def index
+    @tracks = current_user.tracks.where(program: @program)
+  end
+
+  def show; end
+
+  def new
+    @track = @program.tracks.new(color: @conference.next_color_for_collection(:tracks))
+  end
+
+  def edit; end
+
+  def create
+    @track = @program.tracks.new(track_params)
+    @track.submitter = current_user
+    @track.state = 'pending'
```

Comment:
Are you sure you need to set this here? Why don't you keep 'new' as we have in other places as well?